stack/tests/cronjob_test.yaml

```diff
@@ -0,0 +1,35 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/helm-unittest/helm-unittest/main/schema/helm-testsuite.json
+suite: test cronjobs
+templates:
+  - cronjob.yaml
+tests:
+  - it: should work
+    set:
+      cronJobs:
+        job1:
+          concurrencyPolicy: Forbid
+          schedule: "*/3 * * * *"
+          image:
+            repository: my-repo
+            tag: sha-mytag
+          command: ["hello-world"]
```

Comment:
is the binary called hello world? or should this be an arg?

Reply:
this is just a value for the unit test in this context, you are correct that in my changes to argus it needs to be passed into the `args` value